builtin/mainmenu/common.lua

```diff
@@ -300,3 +300,34 @@ function is_server_protocol_compat_or_error(proto_min, proto_max)
 
 	return true
 end
+
+--------------------------------------------------------------------------------
+local function menu_load_worldmt(selected)
```

Comment:
this function is completely pointless. Just use `menudata.worldlist:get_list()[selected]`